dao/src/test/scala/za/co/absa/enceladus/dao/rest/JsonSerializerSuite.scala

```diff
@@ -49,7 +49,7 @@ class JsonSerializerSuite extends BaseTestSuite with VersionedModelMatchers {
           |  "conformance": [],
           |  "parent": null,
           |  "schedule": null,
-          |  "properties": null,
+          |  "properties": {},
```

Comment:
This was also a behavior change, could be subject for discussions

Reply:
My opinion is that we can afford this change since Enceladus 3 going to be a major release anyway.

Reply:
Btw this is due to https://github.com/AbsaOSS/enceladus/pull/2049/files#diff-a1f17980b1a3373c0ebfa182a491b43e99416e11e98c82790c7156249d42f27eR45, I think